webnode/src/utils/encryption.test.js

```diff
@@ -47,3 +48,35 @@ test("nodeForge decryption should produce the expected results", done => {
   }
   done();
 });
+
```

Comment:
The variables in the test below came from an actual upload.

Reply:
Can you put that comment in the file so people know when they read the test in the future?

Reply:
Sure